core/services/keystore/keys/ocr2key/generic_key_bundle.go

```diff
@@ -161,5 +165,11 @@ func (kbraw *keyBundleRawData) Migrate() error {
 		}
 	}
 
+	// if key does not have an ID associated with it (old formats),
+	// derive the key ID and preserve it
+	if bytes.Equal(kbraw.ID[:], new(models.Sha256Hash)[:]) {
```

Comment:
Maybe nbd, but could try to avoid doing this allocation every time.

Reply:
Is there already an empty or zero value somewhere?

Reply:
I didn't find one, but added `models.EmptySha256`